docs/src/lib/manipulation.md

```diff
@@ -20,6 +20,4 @@ join
 melt
 stack
 unstack
-stackdt
```

Comment:
I think I've mentioned this already, but please open a separate PR to remove these.